qutip/entropy.py

```diff
@@ -4,7 +4,8 @@
 
 from numpy import conj, e, inf, imag, inner, real, sort, sqrt
 from numpy.lib.scimath import log, log2
-from . import (ptrace, ket2dm, tensor, sigmay, partial_transpose,
+from qutip import partial_transpose, ket2dm
```

Comment:
maybe better to use `from .partial_transpose import partial_transpose`  and keep ket2dm in the lower input line?